packages/cli/src/databases/repositories/__tests__/folder.repository.test.ts

```diff
@@ -366,6 +366,24 @@ describe('FolderRepository', () => {
 				});
 			});
 
+			it('should return id, name and subFolderCount when specified', async () => {
+				const [folders] = await folderRepository.getManyAndCount({
+					select: {
+						id: true,
+						name: true,
+						subFolderCount: true,
+					},
+				});
+
+				expect(folders).toHaveLength(2);
+				folders.forEach((folder) => {
+					expect(Object.keys(folder).sort()).toEqual(['id', 'name', 'subFolderCount']);
+					expect(folder.id).toBeDefined();
+					expect(folder.name).toBeDefined();
+					expect(folder.subFolderCount).toBeDefined();
```

Comment:
there's only 2 folders in this test, should this rather just test if their property values are what we expect instead of just testing that there's anything, just to make sure the counts get the values we expect?

Reply:
Will address in another PR, else will have to wait for the tests again 💀